doc/cephfs/ceph-dokan.rst

```diff
@@ -8,7 +7,9 @@ Mount CephFS on Windows
 It leverages Dokany, a Windows driver that allows implementing filesystems in
 userspace, pretty much like FUSE.
 
-Please check the `installation guide`_ to get started.
+Please see the `installation guide`_ to get started.
+
+Please see the `OS recommendations`_ regarding client support.
```

Comment:
Noting that the word "package" is missing here, see https://github.com/ceph/ceph/pull/64374#discussion_r2205085101.  I'd suggest fixing that in main and cherry picking that commit into this PR:
```suggestion
Please see the `OS recommendations`_ regarding client package support.
```